### PR TITLE
update dashboards and add alarms provisioning

### PR DESCRIPTION
**Pull request by wkloucek**

### CODE REVIEW

File: config/grafana/dashboards/load-testing/HTTP failures.json [hunk 1]
```diff
@@ -8,19 +8,29 @@
...
+  "id": 3,
+  "iteration": 1642579601387,
   "links": [],
+  "liveNow": false,
   "panels": [
     {
-      "datasource": "K6-InfluxDB-InfluxQL",
+      "datasource": {
+        "type": "influxdb",
+        "uid": "P4D1D31A5B69203FF"
```
Comment (wkloucek) [inline config/grafana/dashboards/load-testing/HTTP failures.json:32]: please note ,that this id also was added in the datasource provisioning file

File: config/grafana/manual-provisioning/contact-points.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "template_files": {
+        "rocketchat - alert": "{{ define \"rocketchatAlert\" }}\n# {{ .Labels.alertname }}\n## Labels\n{{ range .Labels.SortedPairs }}\n- {{ .Name }}: {{ .Value }}\n{{ end }}\n## Annotations\n{{ if gt (len .Annotations) 0 }}\n  {{ range .Annotations.SortedPairs }}\n- {{ .Name }}: {{ .Value }}\n  {{ end }}\n  {{ end }}\n## Links\n  {{ if gt (len .SilenceURL ) 0 }}\n- [Silence alert]({{ .SilenceURL }}) \n  {{ end }}\n  {{ if gt (len .DashboardURL ) 0 }}\n- [Go to dashboard]({{ .DashboardURL }})\n  {{ end }}\n{{ end }}",
+        "rocketchat - messages": "{{ define \"rocketchatMessages\" }}\n  {{ if gt (len .Alerts.Firing) 0 }}\n    {{ range .Alerts.Firing }} {{ template \"rocketchatAlert\" .}} {{ end }}\n  {{ end }}\n  {{ if gt (len .Alerts.Resolved) 0 }}\n    {{ range .Alerts.Resolved }} {{ template \"rocketchatAlert\" .}} {{ end }}\n  {{ end }}\n{{ end }}"
+    },
+    "alertmanager_config": {
+        "route": {
+            "receiver": "jankaritech rocketchat"
```
Comment (individual-it) [inline config/grafana/manual-provisioning/contact-points.json:8]: should we move that already to the owncloud chat?

File: config/grafana/dashboards/load-testing/HTTP failures.json (hunk 1 above)
Comment (individual-it) [reply config/grafana/dashboards/load-testing/HTTP failures.json:32]: Great, so only changing the dashboard file would have actually broken it.

File: config/grafana/manual-provisioning/contact-points.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                "name": "jankaritech rocketchat",
+                "grafana_managed_receiver_configs": [
+                    {
+                        "name": "jankaritech rocketchat",
+                        "type": "slack",
+                        "disableResolveMessage": false,
+                        "settings": {
+                            "recipient": "#test",
+                            "text": "{{ template \"rocketchatMessages\" . }} ",
+                            "token": "foo"
```
Comment (individual-it) [inline config/grafana/manual-provisioning/contact-points.json:25]: this field does not exist in the config I see in the WebUI of grafana I think the authentication purely uses the URL

File: config/grafana/manual-provisioning/contact-points.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        "receivers": [
+            {
+                "name": "jankaritech rocketchat",
+                "grafana_managed_receiver_configs": [
+                    {
+                        "name": "jankaritech rocketchat",
+                        "type": "slack",
+                        "disableResolveMessage": false,
+                        "settings": {
+                            "recipient": "#test",
```
Comment (individual-it) [inline config/grafana/manual-provisioning/contact-points.json:23]: suggesting to use `#ocis-internal`

File: config/grafana/manual-provisioning/contact-points.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                            "text": "{{ template \"rocketchatMessages\" . }} ",
+                            "token": "foo"
+                        },
+                        "secureFields": {
+                            "url": true
```
Comment (individual-it) [inline config/grafana/manual-provisioning/contact-points.json:28]: out of interest: where is that stored?

**Review by individual-it — COMMENTED** [review]
looks great! Thank you!
I made some comments, mainly the question if we want to send the notifications to @owncloud or @JankariTech chat

**Pull request merged**